Tests/RSAEncrypterTests.swift

```diff
@@ -109,7 +109,7 @@ class RSAEncrypterTests: RSACryptoTestCase {
         }
     }
 
-    func testEncryptingWithAliceAndBobKey() {
+    func testEncryptingWithAliceAndBobKey() throws {
```

Comment:
Making tests throw - why did we not think of that in the first place. 🤦‍♂️ Great addition, thanks! 👍 😄